scripts/override_images/override_os_images.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Overrides a given list of OS images to a list with a single OS image
+that matches the latest release image in RELEASE_IMAGES
+"""
+import json
+import os
+import sys
+from argparse import ArgumentParser
+
+from assisted_service_client.models import OsImage, ReleaseImage
+
+from assisted_test_infra.test_infra import utils
+from consts import DEFAULT_CPU_ARCHITECTURE
+
+
+def get_os_image(
+    os_images: list[OsImage], openshift_version: str, cpu_architecture: str = DEFAULT_CPU_ARCHITECTURE
+) -> OsImage | None:
+    archs_images = [v for v in os_images if v.cpu_architecture == cpu_architecture]
+    os_images = [v for v in archs_images if v.openshift_version == openshift_version]
+    if len(os_images) >= 1:
+        return os_images[0]
+
+    return None
+
+
+def extract_version(release_image: str) -> str:
```

Comment:
where are you using this function ?

Reply:
because release images references not always specify the ocp version

Reply:
what? you don't call this function in the script

Reply:
Oh yes I thought you meant `utils.extract_version`